### DISCUSSION TITLE
86E1E9EC-ADF6-C370-CD40-A0C8A151F4E4

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#43] [General]
# 86E1E9EC-ADF6-C370-CD40-A0C8A151F4E4

Kepler-452 is a star similar to the Sun, though slightly more evolved, with a likely super-Earth in the habitable zone. The planet, Kepler-452b, was validated in 2015, but further research in 2018 and 2019 failed to confirm the planet with >99% confidence.

https://celestia.mobi/addon?item=86E1E9EC-ADF6-C370-CD40-A0C8A151F4E4&lang=en

**Comment by exocritic5457** [comment]
I feel like Kepler-452 b would have a similar appearance to Venus, should it exist, since while it orbits in the habitable zone(?) much like Earth, its mass would definitely trap more heat given the higher gravity 

If only NASA wasn't as optimistic with exoplanet habitability.

**Reply by gtfggjbdrff** [reply]
The texture and texture surface of the planet in this plugin are indeed similar to Venus in a few places.

**Comment by gtfggjbdrff** [comment]
这个插件制作的不错！
